openmrs-keycloak-smart-auth/src/main/java/org/openmrs/contrib/keycloak/smart/auth/provider/SmartLaunchAccessAuthenticator.java

```diff
@@ -171,6 +172,13 @@ public void action(AuthenticationFlowContext context) {
 			return;
 		}
 
+		appToken.getOtherClaims()
+				.forEach((key, value) -> {
+					if (value instanceof String) {
+						authSession.setUserSessionNote(SMART_NOTE_PREFIX + key, (String) value);
+					}
+		});
```

Comment:
What is this supposed to be doing exactly? I'm not sure I like the idea of just copying every claim into the `userSessionNote`.

Reply:
@ibacher thanks. Actually, we are coping the 'otherClaims' value (like [PatientUuid](https://github.com/openmrs/openmrs-module-smartonfhir/blob/f2393546a6a4c830085a6305abec77bc3ac8c78a/omod/src/main/java/org/openmrs/module/smartonfhir/web/servlet/SmartAccessConfirmation.java#L62), [VisitUuid](https://github.com/openmrs/openmrs-module-smartonfhir/blob/f2393546a6a4c830085a6305abec77bc3ac8c78a/omod/src/main/java/org/openmrs/module/smartonfhir/web/servlet/SmartAccessConfirmation.java#L65) present into the action token returned from SMART-on-FHIR module) to the `userSessionNote`, so that after the authentication flow end, we can map this parameter into the Access token.